buildenv

```diff
@@ -2,11 +2,14 @@
 PERL_VERSION="5.38.0"
 
 export ZOPEN_BUILD_LINE="STABLE"
+export ZOPEN_STABLE_DEPS="curl gzip make coreutils git make zoslib sed"
+export ZOPEN_STABLE_URL="https://github.com/Perl/perl5/archive/refs/tags/v${PERL_VERSION}.tar.gz"
```

Comment:
Thanks! Another way would be to use the git url and add this line:
```
export ZOPEN_STABLE_TAG="v{PERL_VERSION}"
```

Reply:
Wouldn't that git clone that particular branch? I guess that's ok too, as an alternative to downloading tar.gz?

Reply:
Is STABLE_TAG (git cloning the branch) still needed because the upstreaming work is ongoing?
So making newer patches is easier with the version-controlled clone, as opposed to a zip download at a given version.

Reply:
Either or is fine. I prefer git cloning because we don't have to initialize it as a git repo, as we do for tarballs after extracting.

Reply:
:+1: I've pushed a commit with the tag.